react-frontend/src/components/Home/cardlist.js

```diff
@@ -41,14 +41,15 @@ function CardList() {
                 img={imgLinks[0]}
                 description={descriptions[0]}
                 linkText={linkTexts[0]}
+                routePath="/digital-framework"
               />
             </Col>
             <Col sm={12} md={6}>
               <InfoCard
                 title={'Digital Principles'}
                 img={imgLinks[1]}
                 description={descriptions[1]}
-                linkText={linkTexts[1]}
+                // linkText={linkTexts[1]}
```

Comment:
all of these are commented out.  If they aren't being used they should probably be deleted